deeplake/core/vectorstore/test_deeplake_vectorstore.py

```diff
@@ -2597,3 +2651,5 @@ def test_vectorstore_factory(hub_cloud_dev_token, hub_cloud_path, runtime):
         assert isinstance(db, DeepMemoryVectorStore)
     else:
         assert isinstance(db, DeepLakeVectorStore)
+
```

Comment:
Minor change, to remove these two spaces

Reply:
Done.